ReactApp/src/components/Editor.jsx

```diff
@@ -46,9 +46,7 @@ import { postMessage } from '../misc/Helpers';
 let editor = {};
 
 const POST_MOCK = {
-	id: 1,
```

Comment:
I've removed this due to issues with the API. Since this is not an existing post, the API was receiving this as a parameter and throwing a permissions error.